ecs-cli/modules/clients/aws/cloudformation/client.go

```diff
@@ -238,19 +238,18 @@ func (c *cloudformationClient) waitUntilComplete(stackName string, hasFailed fai
 
 		if successState == status {
 			return nil
+		}
 
-		} else {
-			_, exists := failureStates[status]
-			if exists {
-				log.Debug("Stack creation failed. Getting first failed event")
-				if failureEvent, err := c.firstStackEventWithFailure(stackName, nil, failureStates); err == nil {
-					log.WithFields(log.Fields{
-						"reason":       aws.StringValue(failureEvent.ResourceStatusReason),
-						"resourceType": aws.StringValue(failureEvent.ResourceType),
-					}).Error("Failure event")
-				}
-				return fmt.Errorf("Cloudformation failure waiting for '%s'. State is '%s'", successState, status)
+		_, exists := failureStates[status]
+		if exists {
```

Comment:
I propose `exists` => `failure`. I feel like `if failure` would read more cleanly.

Reply:
that's fair, but I was strictly addressing things that came up through the tool. i can change it though.

Reply:
Actually looking closer at the code, `exists`makes more sense, so I'll keep it as is.